server/src/main/java/keywhiz/service/resources/automation/v2/SecretResource.java

```diff
@@ -482,6 +486,47 @@ public SecretDetailResponseV2 secretInfo(@Auth AutomationClient automationClient
   }
 
   /**
+   * Retrieve contents for a set of secret series
+   *
+   * @excludeParams automationClient
+   *
+   * @responseMessage 200 Secret series information retrieved
+   * @responseMessage 404 Secret series not found
+   */
+  @Timed @ExceptionMetered
+  @POST
```

Comment:
I don't really like having this method as a POST; it doesn't alter the server, only retrieves information from it, which seems closer to a GET.  However, I wanted this to be a bulk endpoint to allow clients to optimize requests to Keywhiz, and I'm not sure how to encode arbitrarily many secret names into the URL's query parameters (or if that's desirable).  One site I found (https://evertpot.com/dropbox-post-api/) mentions a REPORT method for GET-with-request-body situations, but it doesn't seem to be supported by the Javax library we're using for the other annotations.  Since the consensus seems to be that a request body shouldn't be added to GET (http://stackoverflow.com/questions/978061/http-get-with-request-body ), it looks like either making this a POST or having a complex URL are the best options if we want to keep this as a bulk endpoint.